techmatters/flex-plugins
# Chi 1919 ux change category checkmark black

Primary reviewer: @stephenhand 

## Description

- Bug fix: Change category checkmark to black

### Checklist
- [N/A] New tests added
- [N/A] Feature flags added
- [N/A] Strings are localized
- [N/A] Tested for chat contacts
- [N/A] Tested for call contacts

### Related Issues
Fixes CHI-1919

### Verification steps

- This is a simple fix to set the checkmark color to black for all categories. So I removed the color props in the CategoryCheckbox component and set color to a generic black.
- Navigate to Categories in the Tabform, click on a checkbox and confirm that the checkmark color is set to black.
